samples/algorithms/database-search/README.md

```diff
@@ -52,7 +52,7 @@ dotnet run
 ### Running the Sample in Jupyter Notebook
 
 This sample can also be viewed using Jupyter Notebook.
-To do so, ensure that you have the IQ# kernel installed using the instructions in the [getting started guide](https://docs.microsoft.com/azure/quantum/install-jupyter-qkd).
```

Comment:
Ah right, there was a typo in the actual URL for that page that was fixed recently...!